kahuna/public/js/search/index.js

```diff
@@ -82,7 +82,9 @@ search.config(['$stateProvider', '$urlMatcherFactoryProvider',
             const ctrl = this;
 
             if ($stateParams.cropType) {
-                storage.setJs('cropType', $stateParams.cropType, true);
+                $stateParams.cropType === 'all'
```

Comment:
Is there any way to avoid duplicating this code in three places?

Reply:
😢 I thought this would go unnoticed. Yeah, we probably should right?

Reply:
Done.